spec/annotate/annotate_models_spec.rb

```diff
@@ -958,6 +959,7 @@ def self.when_called_with(options = {})
         #  name(ＮＡＭＥ)       :string(50)       not null
         #  notes(ＮＯＴＥＳ)    :text(55)         not null
         #  no_comment           :text(20)         not null
+        #  location             :geometry_collect not null
```

Comment:
This spec will fail with an `ArgumentError: negative argument` if you comment out the newly-added logic in `lib/annotate/annotate_models.rb`